csunplugged/topics/content/en/image-representation/image-representation.md

```diff
@@ -0,0 +1,8 @@
+# Image Representation
+
+Images are everywhere on computers and digital devices, and they are one of the main ways we interact with them.
+Digital devices store everything using digits, so how do they use these digits to create the images we see on our computer screens, our smartphones, and our tablets?
```

Comment:
"how are these digits used" makes more sense, I'll change it to that.

We use "digital devices" so much that it should really have a glossary entry, it can be quite a confusing phrase. I'll add it in at some point, and ask you to review it if that's all good?